pkg/accounting/metrics.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package accounting
+
+import (
+	m "github.com/ethersphere/bee/pkg/metrics"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type metrics struct {
+	// all metrics fields must be exported
+	// to be able to return them by Metrics()
+	// using reflection
+	DebitCount        prometheus.Counter
+	CreditCount       prometheus.Counter
+	DebitEvents       prometheus.Counter
```

Comment:
i'm pretty sure that we don't need these explicitly. this can probably be set up in grafana to display the counter of how many times the debitCount was written to. cc @vandot

Reply:
@acud probably not, Counter holds current value and prometheus scrapes that value every 10s
if you need how many times something happened you have to expose separate metric or for everything you will have 6 occurrences per minute (number of times prometheus scrapes metric)

Reply:
cool. thanks